docs/user/flyout-menu.rst

```diff
@@ -27,31 +33,34 @@ offline formats, and a search bar.
 Customizing the flyout menu
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In your dashboard, you can configure flyout menu options in :guilabel:`Settings`, under :guilabel:`Addons`.
+In your dashboard, you can configure flyout menu options in :guilabel:`Settings > Addons > Flyout Menu`.
 
-Sort your versions :guilabel:`Alphabetically`, by :guilabel:`SemVer`, by :guilabel:`Python Packaging`,
-by :guilabel:`CalVer`, or define your own pattern.
+Version sorting
+^^^^^^^^^^^^^^^
 
-Choose whether to list stable versions first or not.
+The primary customization currently is the ability to sort versions.
+You can sort by:
 
-Customizing the look of the addons flyout menu
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. TODO: Define how these work better..
 
-The addons flyout exposes all the required data to build the flyout menu via a JavaScript ``CustomEvent``.
-Take a look at an example of this approach at https://github.com/readthedocs/sphinx_rtd_theme/pull/1526.
+* :guilabel:`SemVer` - **Default**. As defined by semver.org.
```

Comment:
```suggestion
* :guilabel:`SemVer (Read the Docs)` - **Default**. Read the Docs custom implementation of semver.org.
```

It's important to mention it's a custom implementation, since it supports `v` prefix for example. Also, the name has "Read the Docs" between brackets on purpose because of this.